stable/redis/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 3.8.1
+version: 3.8.2
```

Comment:
Since this is a new feature i think you have to bump to 3.9.0

Reply:
Agree 3.9.0 and I think this one is good to go

Reply:
Works for me!